cryptostore/util.py

```diff
@@ -19,4 +19,7 @@ def get_time_interval(ts: float, interval: str, multiplier=1):
     elif interval == 'D':
         end_interval = dt(timestamp.year, timestamp.month, timestamp.day)
         start_interval = end_interval - timedelta(days=multiplier)
-    return start_interval, end_interval
\ No newline at end of file
+    if start_interval and end_interval:
```

Comment:
time.time() gives seconds since unix epoch, which is in UTC, why do you need to convert them to utc?

Reply:
I wish I had an answer here that I liked. Unfortunately, my answer is that the data being produced was coming out hours off equivalent to my time zone. This fixed that issue.